tests/test_mlbox.py

```diff
@@ -1,28 +1,6 @@
-#!/usr/bin/env python
```

Comment:
Finally, we should remove this test since it is neither a unit test or an integration test

Reply:
ok